src/app/child-dev-project/attendance/attendance-analysis/attendance-analysis.component.ts

```diff
@@ -95,10 +95,11 @@ export class AttendanceAnalysisComponent implements OnInit, AfterViewInit {
 
   applyFilterSelections() {
     this.loading = 0;
-    let filteredData = this.childrenAll;
+    const filteredData = this.childrenAll;
 
+    // TODO reactive filters or rewrite
     this.filterSelections.forEach((f) => {
-      filteredData = filteredData.filter(f.getSelectedFilterFunction());
+      // filteredData = filteredData.filter(f.getSelectedFilterFunction());
```

Comment:
this component will be completely replace due to the event attendance changes. therefore no fixes here